package.json

```diff
@@ -1,43 +1,47 @@
 {
-    "name": "PrairieLearn",
-    "description": "PrairieLearn server app",
-    "version": "2.0.0",
-    "private": true,
-    "dependencies": {
-        "adm-zip": "0.4.x",
-        "archiver": "1.3.x",
-        "async": "2.1.x",
-        "async-stacktrace": "0.0.x",
-        "backbone": "1.3.x",
-        "body-parser": "1.15.x",
-        "cookie-parser": "1.4.x",
-        "copy-dir": "0.3.x",
-        "crypto-js": "3.1.x",
-        "csv": "1.1.x",
-        "ejs": "2.5.x",
-        "express": "4.14.x",
-        "is-my-json-valid": "2.15.x",
-        "javascript-natural-sort": "0.7.x",
-        "jju": "1.3.x",
-        "lodash": "4.17.x",
-        "mongodb": "2.2.x",
-        "node-docker-api": "1.1.x",
-        "numeric": "1.2.x",
-        "pg": "6.1.x",
-        "requirejs": "2.3.x",
-        "serve-favicon": "2.3.x",
-        "socket.io": "1.7.x",
-        "socket.io-client": "1.7.x",
-        "tar-fs": "1.15.x",
-        "winston": "2.3.x"
-    },
-    "devDependencies": {
-        "mocha": "3.2.x",
-        "should": "11.2.x",
-        "request": "2.79.x",
-        "cheerio": "0.22.x"
-    },
-    "scripts": {
-        "test": "node_modules/mocha/bin/mocha tests/index.js"
-    }
+  "name": "PrairieLearn",
```

Comment:
Looks like this is how npm's save works. I always just hand-edit `package.json` so I'd never noticed. In general I don't use `npm --save` because its version specification is not what we want (we lock to minor versions). I suggest we change `package.json` back to 4-space indents and stop using `npm --save` :-)